webapp/channels/src/components/sidebar/__snapshots__/invite_members_button.test.tsx.snap

```diff
@@ -68,17 +68,17 @@ exports[`components/sidebar/invite_members_button should match snapshot 1`] = `
                 "type": [Function],
               }
             }
-            id="introTextInvite"
+            id="inviteMembersButton"
             modalId="invitation"
             onClick={[Function]}
           >
             <button
-              aria-label="Invite Users dialog"
+              aria-label="Invite Members dialog"
               className="style--none intro-links color--link cursor--pointer"
-              id="introTextInvite"
```

Comment:
This fixes the issue of duplicate ID.
<img width="847" alt="Screenshot 2023-07-26 at 5 02 27 PM" src="https://github.com/mattermost/mattermost/assets/5334504/e3cc5ed7-547f-4d1d-b7af-a8ca5fe3f41c">